tests/collections/nlp/test_pretrained_models_performance.py

```diff
@@ -50,51 +50,55 @@ def get_metrics(data_dir, model):
     return metrics
 
 
+def data_exists(data_dir):
+    return os.path.exists(data_dir)
+
+
 class TestPretrainedModelPerformance(TestCase):
     @pytest.mark.unit
```

Comment:
btw. Why this is unit test? You take a pre-trained model and then verify if it is good enough - isn't it an Acceptance Test?